tools/importkey/main.go

```diff
@@ -240,11 +240,12 @@ func main() {
 
 		var err error
 
-		if keyHexString == "" {
+		if keyHexFile == "" {
 			octKey = make([]byte, 32)
 			rand.Read(octKey)
 		} else {
-			octKey, err = hex.DecodeString(keyHexString)
+			// read string from file keyHexFile as hexstring
+			octKey, err = ioutil.ReadFile(keyHexFile)
```

Comment:
ioutil.ReadFile is deprecated, switch to os.ReadFile

Reply:
do you think it would be helpful to replace the other 2 spots in this file that use `ioutil.ReadFile` in this PR?

Reply:
yes